operator/csv-generator/csvtools.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	yaml "github.com/ghodss/yaml"
+	csvv1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// UnmarshalCSV decodes a YAML file, by path, and returns a CSV
+func UnmarshalCSV(filePath string) *csvv1.ClusterServiceVersion {
+	bytes, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		panic(err)
```

Comment:
there are lots of valid reasons why reading a file might fail, which are not system failure warranting a panic. I think this func's signature should include returning errors. Generally, let's follow our best practices and avoid panicking (without handling panics) when the cause is not certainly a system error

Reply:
removed all panics :)